ui.frontend/src/constants.js

```diff
@@ -85,9 +85,24 @@ export const Constants = {
      */
     ARIA_INVALID : "aria-invalid",
 
+    /**
+     * aria attribute to mark the dragged component checked.
+     */
+    ARIA_CHECKED : "aria-checked",
+
     /**
      * Event to trigger when GuideBridge Initialisation Begins
      */
-    GUIDE_BRIDGE_INITIALIZE_START: "bridgeInitializeStart"
+    GUIDE_BRIDGE_INITIALIZE_START: "bridgeInitializeStart",
+
+    /**
+     * attribute to mark the dragged component disabled.
+     */
+    DISABLED : "disabled",
```

Comment:
I have updated these props and added them in an object inside constants. Please pick from that after the checkbox PR gets merged.

cc: @vdua

Reply:
sure

Reply:
When your PR gets merged, I will pull dev branch and resolve this conflict

Reply:
taken pull